src/cc_catalog_airflow/dags/provider_api_scripts/phylopic.py

```diff
@@ -28,12 +26,11 @@
 logger = logging.getLogger(__name__)
 
 
-DELAY = 5
+DELAY = 1
 HOST = 'phylopic.org'
 ENDPOINT = f'http://{HOST}/api/a'
 PROVIDER = 'phylopic'
-FILE = 'phylopic_{}.tsv'.format(int(time.time()))
-LIMIT = 5
+LIMIT = 100
```

Comment:
Since I didn't find a possible rate limit to their API, I preferred to keep `LIMIT` to 100 so as to reduce the number of requests we make. Few large calls or more small calls, What do you think we should go with?

Reply:
In general, the 'fewer large calls' is a good instinct.

In this case, however, there are fewer than 4000 images available from the PhyloPic API.  So, with the delay at 5.0, and limit at 5, we're really making 'few small calls', to be honest.  Let's leave it at that slower rate, unless we have reason to change it in the future.